src/Files.App/Dialogs/CreateArchiveDialog.xaml

```diff
@@ -0,0 +1,160 @@
+﻿<ContentDialog
+	x:Class="Files.App.Dialogs.CreateArchiveDialog"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:helpers="using:Files.App.Helpers"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	Title="{helpers:ResourceString Name=CreateArchive}"
+	d:DesignHeight="300"
+	d:DesignWidth="400"
+	CloseButtonText="{helpers:ResourceString Name=Cancel}"
+	Closing="ContentDialog_Closing"
+	CornerRadius="{StaticResource OverlayCornerRadius}"
+	DefaultButton="Primary"
+	IsPrimaryButtonEnabled="True"
+	Loaded="ContentDialog_Loaded"
+	PrimaryButtonText="{helpers:ResourceString Name=Create}"
+	RequestedTheme="{x:Bind helpers:ThemeHelper.RootTheme}"
+	Style="{StaticResource DefaultContentDialogStyle}"
+	mc:Ignorable="d">
+	<StackPanel Width="440" Spacing="8">
+
+		<!--  Archive Name  -->
+		<Grid
+			Padding="12"
+			Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
+			BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
+			BorderThickness="1"
+			ColumnSpacing="8"
+			CornerRadius="4">
+			<Grid.ColumnDefinitions>
+				<ColumnDefinition Width="*" />
+				<ColumnDefinition Width="Auto" />
+			</Grid.ColumnDefinitions>
+			<TextBlock
+				Grid.Column="0"
+				VerticalAlignment="Center"
+				Text="{helpers:ResourceString Name=Name}" />
+			<TextBox
+				x:Name="FileNameBox"
+				Grid.Column="1"
+				Width="260"
+				PlaceholderText="{helpers:ResourceString Name=EnterName}"
+				Text="{x:Bind ViewModel.FileName, Mode=TwoWay}" />
+		</Grid>
+
+		<!--  Archive Options  -->
+		<Grid
+			Padding="12"
+			Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
+			BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
+			BorderThickness="1"
+			ColumnSpacing="8"
+			CornerRadius="4"
+			RowSpacing="12">
+			<Grid.ColumnDefinitions>
+				<ColumnDefinition Width="*" />
+				<ColumnDefinition Width="Auto" />
+			</Grid.ColumnDefinitions>
+			<Grid.RowDefinitions>
+				<RowDefinition Height="Auto" />
+				<RowDefinition Height="Auto" />
+				<RowDefinition Height="Auto" />
+			</Grid.RowDefinitions>
+
+			<!--  Archive Format  -->
+			<TextBlock
+				Grid.Row="0"
+				Grid.Column="0"
+				VerticalAlignment="Center"
+				Text="{helpers:ResourceString Name=Format}" />
+			<ComboBox
+				x:Name="FileFormatSelector"
+				Grid.Row="0"
+				Grid.Column="1"
+				MinWidth="140"
```

Comment:
Can you hardcode a width instead? `160` is probably enough.

Reply:
It's done.